src/test/test-capability.c

```diff
@@ -103,22 +103,15 @@ static int setup_tests(bool *run_ambient) {
 
         nobody = getpwnam(NOBODY_USER_NAME);
         if (!nobody)
-                return log_error_errno(SYNTHETIC_ERRNO(ENOENT), "Could not find nobody user: %m");
+                return log_warning_errno(SYNTHETIC_ERRNO(ENOENT), "Couldn't find 'nobody' user: %m");
```

Comment:
why downgrade? it's still fatal, no?

Reply:
Define "fatal". We skip the test, but we don't fail it. So I think it's better to only warn.

Reply:
ah, true!